run_pong_no_frameskip_v4.py

```diff
@@ -23,9 +23,12 @@ def parse_args() -> argparse.Namespace:
     parser.add_argument(
         "--cfg-path",
         type=str,
-        default="./configs/pong_no_frameskip_v4/dqn.py",
+        default="./configs/pong_no_frameskip_v4/distillation_dqn.py",
         help="config path",
     )
+    parser.add_argument(
```

Comment:
I think Lunarlander also need buffer path flag.
Or distillation configs and option of lunarlander should be removed.